cms/templates/cms/program_page.html

```diff
@@ -58,7 +58,7 @@ <h1 class="title">
         <div class="description">
           {{ page.title_over_image|richtext }}
         </div>
-        {% if courses %}
+        {% if not page.program_subscribe_link %}
           {% if not authenticated %}
             <button class="mdl-button hero-button open-signup-dialog mdl-cell--hide-phone">
               Sign Up Now
```

Comment:
I think we're going to get a merge conflict with #4868. Would you mind merging that first and then rebasing this?